docs/pages/careers/react-engineer-docs-infra.md

```diff
@@ -0,0 +1,113 @@
+# React Engineer — Docs Infra
+
+<p class="description">You will drive the development of the documentation platform that powers all MUI products, enabling it to grow and establish itself as a mature product.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-5 to UTC+2).
```

Comment:
Reduced the timezone preference a bit from UTC-6/UTC+5 to UTC-5/UTC+2 so it overlaps more with me, making it easy to have meetings and fast development.